README.md

```diff
@@ -87,7 +87,7 @@ export default MyStoreCheckout;
 
 ### Setting up your payment form (`injectStripe`)
 
-Use the `injectStripe` HOC to build your payment form components in the `Elements` tree. This HOC injects the `stripe`
+Use the `injectStripe` Higher Order Component (HOC) to build your payment form components in the `Elements` tree. This HOC injects the `stripe`
```

Comment:
I think it should actually be dasherized: https://facebook.github.io/react/docs/higher-order-components.html